Runtime/Scripts/Rooms/AsyncInstractions/PublishTrackInstruction.cs

```diff
@@ -25,6 +25,9 @@ void OnPublish(PublishTrackCallback e)
             IsError = !string.IsNullOrEmpty(e.Error);
             IsDone = true;
 
+            if (!IsError)
+                _room.OnLocalTrackPublished(e.Publication);
```

Comment:
Do we need this here?  Shouldn't we go off of LocalTrackPublished MessageOneofCase event in Room?